packages/spark-core/base/_links.scss

```diff
@@ -18,21 +18,21 @@
   &:hover,
   &:active,
   &:focus {
-    color: $link-underline-color;
-    border-bottom: $link-underline-width solid $link-underline-color;
+    color: $link-hover-color;
+    border-bottom: $link-underline-width solid $link-hover-color;
   }
 }
 
 .sprk-b-Link--standalone {
   color: $link-standalone-color;
-  border: $link-underline-width solid transparent;
+  border-bottom: $link-underline-width solid transparent;
   font-weight: $link-standalone-font-weight;
 
   &:hover,
   &:active,
   &:focus {
     transition: $link-transition;
```

Comment:
Hey @bobwise, can you add a var here so people can change the link color on hover?